debugger.hpp

```diff
@@ -69,7 +69,7 @@ inline void debug_ast(AST_Node* node, string ind = "", Env* env = 0)
         case Complex_Selector::ADJACENT_TO: cerr << "{+}"; break;
```

Comment:
I saw you post debub_ast output in an issue and fell in love.
On 28 Mar 2015 23:05, "Marcel Greter" <notifications@github.com> wrote:

> In debugger.hpp
> <https://github.com/sass/libsass/pull/986#discussion_r27344568>:
>
> > @@ -69,7 +69,7 @@ inline void debug_ast(AST_Node* node, string ind = "", Env* env = 0)
> >          case Complex_Selector::ADJACENT_TO: cerr << "{+}"; break;
>
> Looks like your using debug_ast too [image: :wink:] I hope you like it as
> much as I do [image: :smile:] It actually decreased my time hunting bugs
> dramatically! I also thought about creating a nicer debug_env function,
> but I was able to use existing one (altough the output could be better
> there).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sass/libsass/pull/986/files#r27344568>.
>